mesop/components/link/link.scss

```diff
@@ -0,0 +1,3 @@
+a {
+  display: block;
```

Comment:
Curious on the decision to make this a block element by default

Reply:
I was on the fence because link is normally inline. But because text is a block element, it seems good to be consistent between text and link since link is kind of like a text.

There's also something about writing UIs in Mesop/Python that makes it seem more natural to treat these things as blocks because it resembles the code structure, for example:

```py
me.text("before")
me.link(...)
me.text("after
```

vs. in HTML, it's common to put it all on a single line:

```html
<div>before<a>link</a>after</div>
```

This is the part I've been unsure about because things like <img> for example are also inline by default and I've left it as inline. WDYT?

Reply:
I think because the text component is block, I think link as block is ok since if the user wanted to to make them flow together, they'd have to change both. 

I don't feel strongly either way.